backend/tracim_backend/lib/core/content.py

```diff
@@ -414,6 +415,91 @@ def get_base_query(
     # 
     #     return result
 
+    def _is_content_label_is_free(
```

Comment:
`_is_content_label_is_free` to be renamed `_is_content_label_free` or `_content_label_is_free`